### PR TITLE
Proper credits references

### PR DESCRIPTION
**Pull request by Tom-van-Woudenberg**
Following our discussion, this pull request includes several changes to the TeachBooks manual, focusing on adding references, updating the table of contents, and improving the credits and contact information

What do you think?

Below are the most important changes:

# References and Bibliography: https://teachbooks.io/manual/credits_references/references.html

* Added a bibliography file `references.bib` and included it in the configuration (`book/_config.yml`).
* Created a new `references.md` file to display the bibliography and linked external resources.
* Added a citation for "LaTeX to Markdown conversion" in the `references.bib` file.

## Credits and Contact Information: https://teachbooks.io/manual/credits_references/credits.html

* Added a new `credits.md` file to provide detailed credits, licensing information, and contact details.
* Removed contact and contribution information from `intro.md` as it has been moved to `credits.md`.

### CODE REVIEW

**Comment by FreekPols** [comment]
Lastige is dat inhoud ook veelal genaakt wordt door TA's. Maar volgens mij kun je de auteur zien als de inhoudelijke eigenaar / verantwoordelijke. 

Bibtex van de teachbooks kan direct opgenomen worden in de tekst.

File: book/references.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# References
+:::{bibliography}
+:::
+
+## External recourses
+
+Parts of this book are taken from other external resources. The following are not edited by the TeachBooks Team:
+
+- [](external/latex-to-markdown-conversion/README.md) {cite:ts}`timon_latex`, not licensed.
```
Comment (rlanzafame) [inline book/references.md]: technically if it is not licensed we cannot reuse it. we should ask timon to put a license on it, or get permission to use it.

**Comment by rlanzafame** [comment]
> Lastige is dat inhoud ook veelal genaakt wordt door TA's. Maar volgens mij kun je de auteur zien als de inhoudelijke eigenaar / verantwoordelijke.
> 
> Bibtex van de teachbooks kan direct opgenomen worden in de tekst.

I think as a general rule TA's adding content does not make them authors, but Julie seems to be an exception, as even though she is hired as a TA she has put a significant amount of creative input into the structure and contents of the manual. @Tom-van-Woudenberg ?

**Comment by rlanzafame** [comment]
It seems to me the references page is being used as an acknowledgements/credits page. The acknowledgements/credits page should be the place where we indicate where and how specific materials are used. References should only be a list of the sources referred to in the text (a la a scientific article). There is of course going to be some overlap, but I would think it is easier to keep track of this by making sure re-used material used is explicitly listed somewhere. For example:

1. include a figure from someone else in your book? the text includes a citation when it is being explained, but the credits page lists that the figure is reused.
2. explanation of a complex phenomenon from another book included verbatim in a page? list it on the credits page (and i'd probably also put a commented tag in the source code of my page delimiting the quote). Depending on the context of the other pages i'd also maybe note that the page includes material from someone else (with the quicklink to the credits page)
3. if i also wanted to describe the relevance and important of the quote above and refer readers to the original book, then i would cite it also, and only then i would add to references.

agree? disagree?

File: book/references.md (hunk 1 above)
Comment (Tom-van-Woudenberg) [reply book/references.md]: I agree

**Comment by Tom-van-Woudenberg** [comment]
> I think as a general rule TA's adding content does not make them authors, but Julie seems to be an exception, as even though she is hired as a TA she has put a significant amount of creative input into the structure and contents of the manual. @Tom-van-Woudenberg ?

All content is from TU Delft, in that sense authors are not difference than TAs. But Julie for sure is an exception

**Comment by Tom-van-Woudenberg** [comment]
For proper APA referencing, now did some ugly stuff. @rlanzafame will fix this later. See #35

**Review by rlanzafame — APPROVED** [review]

**Pull request merged**